src/openapi/peridio-admin-openapi.yaml

```diff
@@ -22,6 +22,26 @@ x-tagGroups:
       - "Products"
       - "Users"
 paths:
+  /orgs/{organization_name}/ca_certificates/verification_codes:
```

Comment:
Please move this block to be right before the `/orgs/{organization_name}/keys:` block.

Reply:
solved!